cmd/frontend/graphqlbackend/search_results.go

```diff
@@ -659,25 +660,32 @@ func (r *searchResolver) evaluateLeaf(ctx context.Context) (*SearchResultsResolv
 		trace.GraphQLRequestName(ctx),
 	).Inc()
 
-	if v := conf.Get().ObservabilityLogSlowSearches; v != 0 && time.Since(start).Milliseconds() > int64(v) {
-		// Note: We don't care about the error here, we just extract the username if
-		// we get a non-nil user object.
-		currentUser, _ := CurrentUser(ctx)
-		var currentUserName string
-		if currentUser != nil {
-			currentUserName = currentUser.Username()
-		}
-
-		log15.Warn("slow search request",
-			"time", time.Since(start),
-			"query", `"`+r.rawQuery()+`"`,
-			"type", trace.GraphQLRequestName(ctx),
-			"user", currentUserName,
-			"source", trace.RequestSource(ctx),
-			"status", status,
-			"alertType", alertType,
-		)
+	isSlow := time.Since(start) > logSlowSearchesThreshold()
+	if honey.Enabled() || isSlow {
+		var act actor.Actor
+		if a := actor.FromContext(ctx); a != nil {
+			act = *a
+		}
+
+		ev := honey.Event("search")
+		ev.AddField("query", r.rawQuery())
+		ev.AddField("actor_uid", act.UID)
+		ev.AddField("actor_internal", act.Internal)
+		ev.AddField("type", trace.GraphQLRequestName(ctx))
+		ev.AddField("source", string(trace.RequestSource(ctx)))
+		ev.AddField("status", status)
+		ev.AddField("alertType", alertType)
```

Comment:
```suggestion
		ev.AddField("alert_type", alertType)
```

(unless mixed camel/snake case is intentional 😞 )

Reply:
hold over from slow log logging. Prefer not snake in this case. thanks